### DISCUSSION TITLE
Railgun ammo rack should hold 5 shells

### DISCUSSION THREAD

**Discussion by leonard84** [FakeFishGames/Barotrauma#8431] [Ideas and suggestions]
The current railgun rack can only hold three rounds but takes up a lot of space and has quite a lot of space between the shells. When we compare this to the loader which somehow manages to hold 5 shells in half the space (probably by stacking them) it is a bit ridiculous.

There are already [mods](https://steamcommunity.com/sharedfiles/filedetails/?id=1953240552) addressing this issue, but IMHO this should be changed in vanilla.

**Comment by GoodGuyJimmy** [comment]
Honestly, you're right
